src/content/blog/web-framework-performance-report.mdx

```diff
@@ -0,0 +1,132 @@
+
+# Core Web Vitals
+
+[Google’s Core Web Vitals (CWV)](https://web.dev/learn-core-web-vitals/) are a set of three standardized metrics that help you understand how users experience a web page. Each metric measures a different aspect of user experience — load speed, responsiveness, visual stability — and together they quantify the overall performance of a website.
+
+Google’s [Core Web Vitals Assessment](https://pagespeed.web.dev/) is a test that looks at real user measurement data (from the [CrUX](https://developer.chrome.com/docs/crux/) dataset) across all three metrics to determine an overall pass/fail grade for each website. For a website to pass, it must meet the [associated threshold](https://web.dev/defining-core-web-vitals-thresholds/) of **“good”** in all three metrics. If any metric fails the threshold, the website fails the assessment.
+
+The CWV Assessment is unique in it's use of real-world user data and measurements. This makes it a more accurate reflection of how users actually experience a website, especially over longer sessions. Lighthouse and other lab testing tools are only able to measure first page load.
+
+<CWVCulminativeChart />
+
+When looking at all known websites built with a certain framework, Astro and SvelteKit beat the average pass rate of all websites tested (40.5%) while the rest of the frameworks did not. Astro was the only framework to reach above 50% of websites passing Google's CWV Assessment. Next.js and Nuxt came in at the bottom of the pack with roughly 1-in-4 and 1-in-5 websites passing the assessment, respectively.
+
+What is the most likely cause for a website to fail Google's Core Web Vitals Assessment? We can break down the data by individual metric to gain insight into where different frameworks struggle (and succeed) when it comes to web vitals.
+
+<CWVFIDChart />
+
+[First Input Delay (FID)](https://web.dev/fid/) measures the time from when a user first interacts with a page to the time when the browser is able to respond to that interaction. Google's CWV Assessment looks for a FID of 100 milliseconds or less. Anything slower is considered needing improvement and failing the assessment.
+
+Most of our frameworks pass this test handily, with over 90% or more of websites passing the assessment. No framework drops below an 80% pass rate on this test. This means that most of the websites tested are responsive to the first user interaction.
+
+<CWVCLSChart />
+
+[Cumulative Layout Shift (CLS)](https://web.dev/cls/) measures visual stability on the page. To pass this assessment, you should reduce unexpected layout shift to near-zero to give your users a reliable visual experience.
+
+CLS is an interesting metric for Google to include as one of the three Core Web Vitals because it isn't strictly related to speed or responsiveness. It's inclusion underscores the importance of looking at more than just performance when it comes to measuring the overall quality of user experiences on the web.
```

Comment:
Typo: `it's` -> `its`
```suggestion
CLS is an interesting metric for Google to include as one of the three Core Web Vitals because it isn't strictly related to speed or responsiveness. Its inclusion underscores the importance of looking at more than just performance when it comes to measuring the overall quality of user experiences on the web.
```